src/dmd/dsymbolsem.d

```diff
@@ -86,6 +86,13 @@ private extern (C++) FuncDeclaration buildPostBlit(StructDeclaration sd, Scope*
     if (sd.isUnionDeclaration())
         return null;
 
+    bool hasCopyCtor = sd.copyCtor !is null;
```

Comment:
`hasCopyCtor` seems pointless. Just `if (sc.copyCtor)` will work fine.

Reply:
It's not. It's used again at line 297

Reply:
This looks like a dead variable to me.

Reply:
Now it is, but it wasn't